frontend/src/lib/Quiver.ts

```diff
@@ -232,7 +232,15 @@ interface DataFrameCell {
   content: DataType
 
   /** The cell's content type. */
+  // NOTE: `DataTypeName` should be used here, but as it's hard (maybe impossible)
+  // to define such recursive types in TS, `string` will suffice for now.
+  // For "blank" cells "contentType" is undefined.
+  // For "columns" cells "contenType" is always set to "unicode"
+  // (ArrowJS automatically converts them to strings).
   contentType?: string
+
+  /** The cell's Styler displayed value. */
+  displayContent?: string
```

Comment:
Alternate, possibly clearer docstring for displayContent:

```
/** 
 * The cell's formatted content string, if the DataFrame was created with a Styler.
 * If the DataFrame is unstyled, displayContent will be undefined, and display
 * code should apply a default formatting to the `content` value instead.
 */
```

Also, is this new field used anywhere? (Shouldn't `BetaTable.tsx` be changed to use it?)

Reply:
Done